src/Nav.js

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
```

Comment:
this is boilerplat from MUI?

Reply:
Yeah it is. Thats one of the things I wanted to ask you about. I've never seen it before. Looks pretty important though.